avm/server/src/lib.rs

```diff
@@ -46,7 +45,13 @@ pub use fluence_faas::HostImportError;
 pub use fluence_faas::IType;
 pub use fluence_faas::IValue;
 
+pub use air_interpreter_interface::CallRequests;
+pub use air_interpreter_interface::CallResults;
 pub use air_interpreter_interface::InterpreterOutcome;
```

Comment:
Is there any reason to keep InterpreterOutcome in exports of avm-server? I'd think AVMOutcome supersedes it

Reply:
Nope, you're right.